sgx/src/lib.rs

```diff
@@ -9,6 +9,72 @@
 #![allow(clippy::identity_op)]
 #![deny(missing_docs)]
 
+#[cfg(test)]
+#[macro_use]
+mod testing {
```

Comment:
I don't see any reason to have the macro in its own module at this point.

Reply:
Excellent point. Module removed!